### DISCUSSION TITLE
Dexcom G6 + Airplane Mode = Sensor Fail?

### DISCUSSION THREAD

**Discussion by jw243** [NightscoutFoundation/xDrip#2318] [Q&A]
I tried my first Dexcom G6 sensor as part of the "Hello Dexcom" Scheme.
I followed [this guideline](https://navid200.github.io/xDrip/docs/G6-Recommended-Settings.html) to start the sensor and everything seemed to work as it should at first.
However, during the warmup period of the sensor, my phone went into **airplane mode** for 1,5 hours and the bluetooth connection was cut off. Upon reconnection, the sensor was **marked as failed** and could not be started anymore, neither by XDrip, nor by the Dexcom app.

Therefore, I would be interested to know if anyone else has experienced a similar behaviour:
- Will a sensor always be marked as failed if bluetooth connection is lost during sensor warmup?
- Or is there a maximum time that bluetooth can be disconnected during warmup without causing a sensor failure?
- Or was the sensor faulty in the first place and the failure had nothing to do with XDrip?
- Can longer bluetooth disconnections generally lead to sensor failures, even after the warmup period?

Any experience in this regard would be greatly appreciated. Thanks.

**Comment by Navid200** [comment]
Bluetooth disconnect cannot cause a sensor to fail.  You can start a sensor and then turn off your phone, which means there will be no Bluetooth connectivity as far as the transmitter is concerned.  If you connect later, everything will be fine.

However, there are factors that can cause a sensor session to fail.
As an example, if the transmitter is not disconnected from your body for at least 10 minutes while the sensor is stopped between the two consecutive sessions, the transmitter may trigger a restart attempt flag and stop the session.

The link you have posted lists the recommended settings.
These are the guidelines for starting:  https://navid200.github.io/xDrip/docs/Starting-G6.html

Now that you have started already and it failed, your only option is to perform a restart:  https://navid200.github.io/xDrip/docs/Restart-G6-sensor.html

**Reply by jw243** [reply]
Thanks for your quick answer. It's good to hear that bluetooth disconnection was not the culprit.

Since it was my first Dexcom sensor, the transmitter had never been connected to any other sensor. So this could also not be the reason for the sensor fail.

I tried to restart the sensor by pulling out the transmitter, waiting for 30 minutes and popping it back in. But the sensor would't start again, neither with XDrip nor with the Dexcom app.

**Reply by Navid200** [reply]
Without even seeing a screenshot of the system status page, and only looking at what you have posted here, it sounds like a connectivity issue.

**Reply by jw243** [reply]
Could you explain what you mean by "connectivity issue"? The transmitter connected to xDrip as well as the Dexcom app without any problems.

However, the sensor itself failed both in xDrip and the Dexcom app during warmup.

From all the information I have, I guess that the sensor was indeed faulty.

**Reply by Navid200** [reply]
This is not a very simple system.  You cannot use both xDrip and Dexcom app at the same time as collectors from the same transmitter.

Just because you have connectivity for one read cycle, it doesn't mean that everything is perfectly working.
